motioneye/templates/main.html

```diff
@@ -161,6 +163,15 @@
                         <span class="minimize open"></span>
                     </div>
                     <table class="settings">
+                        <tr class="settings-item" strip="true">
+                            <td class="settings-item-label"><span class="settings-item-label">{{ _("Lingvo") }}</span></td>
+                            <td class="settings-item-value"><select class="styled general main-config" id="langSelect">
+                              {% for lang in settings.langlist %}
```

Comment:
Ah the moment the list is empty. Not sure whether `settings` is available when the HTML is generated? Otherwise probably we just add a static list? Since language won't be added or removed often, it wouldn't be so hard to maintain.

Reply:
settings.langlist is populated in meyectl.load_l10n(), which is called before server.main(), so it's available when the HTML is generated.
Here is the result :
![image](https://user-images.githubusercontent.com/48726908/167012723-80e0e737-e958-4fbd-8a80-01c1fc839403.png)